lib/opentracing.rb

```diff
@@ -0,0 +1,48 @@
+require "forwardable"
+require "concurrent"
+require "opentracing/version"
+require "opentracing/span_context"
+require "opentracing/span"
+require "opentracing/nil_tracer"
+
+module OpenTracing
+  FORMAT_TEXT_MAP = 1
```

Comment:
OK. Also, I put inject and extract on the top level OpenTracing. Should that be moved?